cloud/team/team.go

```diff
@@ -0,0 +1,708 @@
+	if err != nil {
+		return err
+	}
+	err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(out, "The workspace team %s role was successfully updated to %s\n", teamID, role)
+	return nil
+}
+
+func UpdateTeam(id, name, description string, out io.Writer, client astrocore.CoreClient) error {
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+	if ctx.OrganizationShortName == "" {
+		return user.ErrNoShortName
+	}
+	teams, err := GetOrgTeams(client)
+	if err != nil {
+		return err
+	}
+	if len(teams) == 0 {
+		return ErrNoTeamsFoundInOrg
+	}
+	var team astrocore.Team
+	if id == "" {
+		team, err = selectTeam(teams)
+		if err != nil {
+			return err
+		}
+	} else {
+		for i := range teams {
+			if teams[i].Id == id {
+				team = teams[i]
+			}
+		}
+		if team.Id == "" {
+			return ErrTeamNotFound
+		}
+	}
+	teamID := team.Id
+
+	teamUpdateRequest := astrocore.UpdateTeamJSONRequestBody{}
+
+	if name == "" {
+		teamUpdateRequest.Name = team.Name
+	} else {
+		teamUpdateRequest.Name = name
+	}
+
+	if description == "" {
+		teamUpdateRequest.Description = *team.Description
+	} else {
+		teamUpdateRequest.Description = description
+	}
+
+	resp, err := client.UpdateTeamWithResponse(httpContext.Background(), ctx.OrganizationShortName, teamID, teamUpdateRequest)
+	if err != nil {
+		return err
+	}
+	err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(out, "Astro Team %s was successfully updated\n", team.Name)
+	return nil
+}
+
+func RemoveWorkspaceTeam(id, workspace string, out io.Writer, client astrocore.CoreClient) error {
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+	if ctx.OrganizationShortName == "" {
+		return user.ErrNoShortName
+	}
+	if workspace == "" {
+		workspace = ctx.Workspace
+	}
+	teams, err := GetWorkspaceTeams(client, workspace, teamPagnationLimit)
+	if err != nil {
+		return err
+	}
+	if len(teams) == 0 {
+		return ErrNoTeamsFoundInWorkspace
+	}
+	var team astrocore.Team
+	if id == "" {
+		team, err = selectTeam(teams)
+		if err != nil {
+			return err
+		}
+	} else {
+		for i := range teams {
+			if teams[i].Id == id {
+				team = teams[i]
+			}
+		}
+		if team.Id == "" {
+			return ErrTeamNotFound
+		}
+	}
+	teamID := team.Id
+	resp, err := client.DeleteWorkspaceTeamWithResponse(httpContext.Background(), ctx.OrganizationShortName, ctx.Workspace, teamID)
+	if err != nil {
+		return err
+	}
+	err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(out, "Astro Team %s was successfully removed from workspace %s\n", team.Name, ctx.Workspace)
+	return nil
+}
+
+func selectTeam(teams []astrocore.Team) (astrocore.Team, error) {
+	table := printutil.Table{
+		Padding:        []int{30, 50, 10, 50, 10, 10, 10},
```

Comment:
nit: do we need all these paddings, since we have just 3 columns

Reply:
looks like when Dynamic Padding is true these values are ignored.